pkg/config/vault.go

```diff
@@ -369,7 +369,8 @@ func RemoveVaultSecretFiles() {
 func createTemporarySecretFile(namePattern string, content string) (string, error) {
 	if VaultSecretFileDirectory == "" {
 		var err error
-		VaultSecretFileDirectory, err = ioutil.TempDir(".", "vault")
+		fileUtils := &piperutils.Files{}
+		VaultSecretFileDirectory, err = fileUtils.TempDir("", "vault")
```

Comment:
will this try to create secret file in the root directory ?

Reply:
there is some additional logic in our fileUtils.
If I recall it correctly, then this was introduced with a similar problem during kanikoExecute ...

Reply:
yes : https://github.com/SAP/jenkins-library/blob/a610e1df6a239bcf6d65ff7749de9ee2b4877a85/pkg/piperutils/FileUtils.go#L54 should take care